src/main/kotlin/io/kotest/plugin/intellij/toolwindow/tags.kt

```diff
@@ -0,0 +1,56 @@
+package io.kotest.plugin.intellij.toolwindow
+
+import com.intellij.openapi.project.DumbService
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiTreeAnyChangeAbstractAdapter
+import io.kotest.plugin.intellij.findFiles
+import io.kotest.plugin.intellij.psi.getAllSuperClasses
+import org.jetbrains.kotlin.idea.core.util.toPsiFile
+import org.jetbrains.kotlin.name.FqName
+import org.jetbrains.kotlin.psi.KtClassOrObject
+import org.jetbrains.kotlin.psi.KtProperty
+
+const val TagsFilename = "KotestTags.kt"
```

Comment:
I suppose scanning all files would be too heavy on performance?

Reply:
Yes, and also we could run into "matches" which weren't really tags.
We can always start here and expand it.